extension/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
```

Comment:
well there's the compile step itself, and everything with babel in the name in here, and the npm dependencies... :P

but if it comes from the generator that may just be annoying to remove